### PR TITLE
feat(sdk-core): add encryptedWalletPassphrase to generateWallet method

### PR DESCRIPTION
**Pull request by alebusse**
Added a new field in the generateWallet response that its the wallet passphrase encrypted with the passcode, this data can be used to do recover the original password, same as the box D on a keycard. Its only available for hot wallets and if passphrase and the passcode is provided

WP-2525

TICKET: WP-2525

### CODE REVIEW

File: modules/bitgo/test/v2/unit/wallets.ts [hunk 1]
```diff
@@ -490,15 +528,64 @@ describe('V2 Wallets:', function () {
 
       const wallets = new Wallets(bitgo, tsol);
 
-      await wallets.generateWallet({
+      const params = {
         label: 'tss wallet',
         passphrase: 'tss password',
-        multisigType: 'tss',
+        multisigType: 'tss' as any,
```
Comment (sachushaji) [inline modules/bitgo/test/v2/unit/wallets.ts:534]: Can't we avoid this any type casting?, multisigType should consider tss as valid

**Review by sachushaji — DISMISSED** [review]

File: modules/bitgo/test/v2/unit/wallets.ts (hunk 1 above)
Comment (OttoAllmendinger) [reply modules/bitgo/test/v2/unit/wallets.ts:534]: pretty sure `as const` will work here

**Review by OttoAllmendinger — DISMISSED** [review]

**Review by margueriteblair — COMMENTED** [review]
can approve once the casting to `any` comments are addressed

**Review by margueriteblair — DISMISSED** [review]

File: modules/sdk-core/src/bitgo/wallet/wallets.ts [hunk 2]
```diff
@@ -1008,4 +1024,14 @@ export class Wallets implements IWallets {
...
+
+  /**
+   * Returns the encrypted wallet password
+   * @param {string} passphrase - the passphrase to encrypt
+   * @param {string}passcodeEncryptionCode - the passcode encryption code
+   * @returns {string} - the encrypted wallet password
+   */
+  private getEncryptedWalletPassphrase(passphrase: string, passcodeEncryptionCode: string): string {
+    return this.bitgo.encrypt({ input: passphrase, password: passcodeEncryptionCode });
+  }
```
Comment (zahin-mohammad) [inline modules/sdk-core/src/bitgo/wallet/wallets.ts:1036]: Do we really need this? We can just directly call `this.bitgo.encrypt`

**Review by zahin-mohammad — DISMISSED** [review]

**Review by zahin-mohammad — APPROVED** [review]

**Review by margueriteblair — APPROVED** [review]

**Pull request merged**